kafka/build_darwin_amd64.go

```diff
@@ -1,6 +1,6 @@
+//go:build !dynamic
```

Comment:
Don't modify this file. I know go fmt modifies this automatically but since our commitment is to support as old a version of go as we possibly can, we avoid it. Same for the other build_ files.
Eventually, one day, we will make the change but not today.

Reply:
sure I will checkout all the build files from the base branch commit.